src/react/utils.js

```diff
@@ -91,6 +91,20 @@ export function isReactElement(val: Value): boolean {
   return false;
 }
 
+export function isReactProps(val: Value): boolean {
```

Comment:
Perhaps call this "inReactProps"? Come to think of it, reactProps is not a very helpful name for someone unfamiliar with the code base.

Reply:
Maybe `isReactPropsObject`? A single object is called "props", it's not a plural in this sense.

The fact that it checks a set is an implementation detail. It's meant to be more of a brand check conceptually. Like `isArray` rather than `inArrays`.